dictionary.json

```diff
@@ -3,6 +3,70 @@
   "description": "The Attribute Dictionary defines attributes and includes references to the events and objects in which they are used.",
   "name": "dictionary",
   "attributes": {
+    "activities": {
+      "type": "activities",
+      "caption": "Activities"
+    },
+    "authentication_activities": {
+      "caption": "Authentication Activities",
+      "description": "Authentication Activities events report authentication session activity.",
+      "is_array": true,
+      "type": "authentication_related_activity"
+    },
+    "dns_activities": {
+      "caption": "DNS Activities",
+      "description": "DNS Activities report DNS queries and answers activity.",
+      "is_array": true,
+      "type": "dns_related_activity"
+    },
+    "email_activities": {
+      "caption": "Email Activities",
+      "description": "Email Activity reports mail activities",
+      "is_array": true,
+      "type": "email_related_activity"
+    },
+    "filesystem_activities": {
```

Comment:
Even though the long form, Pascal case full name of the original activity is `File System Activity`, and the filename is `filesystem.json`, the snake case short form is `file_activity` instead of `filesystem_activity`. So I think for consistency, the short form here should also be `file_activities`.